metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -168,26 +164,17 @@ def s3_client(self):
     @validator("glue_s3_lineage_direction")
     def check_direction(cls, v: str) -> str:
         if v.lower() not in ["upstream", "downstream"]:
-            raise ConfigurationError(
+            raise ValueError(
```

Comment:
How come you swap to ValueError here? Should all config validators raise ValueErrors?

Reply:
yes - with pydantic, you're supposed to raise ValueError or AssertionError

that way pydantic can (1) show if there are multiple validation issues and (2) shows the exact fields that failed validation alongside the errors